packages/react-composites/src/composites/CallWithChatComposite/EmbeddedChatPane.tsx

```diff
@@ -16,17 +18,34 @@ import { FileSharingOptions } from '../ChatComposite';
 export const EmbeddedChatPane = (props: {
   chatCompositeProps: Partial<ChatCompositeProps>;
   chatAdapter: ChatAdapter;
+  callAdapter: CallAdapter;
   fluentTheme?: PartialTheme | Theme;
   hidden: boolean;
   onClose: () => void;
   onFetchAvatarPersonaData?: AvatarPersonaDataCallback;
   onChatButtonClick: () => void;
   onPeopleButtonClick: () => void;
+  modalLayerHostId: string;
   mobileView?: boolean;
   /* @conditional-compile-remove(file-sharing) */
   fileSharing?: FileSharingOptions;
 }): JSX.Element => {
   const callWithChatStrings = useCallWithChatCompositeStrings();
+  const theme = useTheme();
+
+  const pipStyles: WithLocalAndRemotePIPStyles = useMemo(
+    () => ({
+      modal: {
+        main: {
+          top: '3.25rem',
```

Comment:
For this:

> Bounds of Modal is not working when the starting position is set through styling such as { main: { top: '3.25rem' } }

Seems like a bad bug. Why do we need to set it for ChatPane and not for PeoplePane?

Reply:
No longer using the `main.top` style property so the [bad bug](https://github.com/microsoft/fluentui/issues/20122) shouldn't be an issue now.